src/Export/Adapters/PropertiesAdapter.php

```diff
@@ -14,7 +14,7 @@
 use Shopware\Core\System\SalesChannel\SalesChannelContext;
 use Symfony\Contracts\Translation\TranslatorInterface;
 
-class PropertiesAdapter
+class PropertiesAdapter extends ShopwarePropertiesAdapter
```

Comment:
To make it more clear, I would call `PropertiesAdapter` something like `DefaultPropertiesAdapter`